wp-sw-cache/lib/service-worker.js

```diff
@@ -1,61 +1,65 @@
-var CACHE_NAME = '$name';
+(function(self){
+  var CACHE_PREFIX = '__wp-sw-cache::';
 
-var CACHE_FILES = $files;
+  var CACHE_NAME = CACHE_PREFIX + $name;
```

Comment:
I added the prefix to prevent you from deleting my DBs. And I should do the same, by the way. :P